### PR TITLE
Revert "Validation Feature"

### PR DESCRIPTION
**Pull request by donice**
I opened two pull requests https://github.com/donice/Microverse-Portfolio/pull/8 and https://github.com/donice/Microverse-Portfolio/pull/9. The latter was approved but the former wasn't. I merged but it seemed to have automatically merged both.

I've reverted the Pull request, but it's not opened in https://github.com/donice/Microverse-Portfolio/pull/10. I can no longer see my code reviews but I see ‘Changes are required’ in my dashboard.

Please kindly revert here for previous comments Reverts donice/Microverse-Portfolio#8

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -118,17 +116,25 @@ renderProjects();
...
   const data = projects[index - 1];
-  const [technology1, technology2, technology3] = data.technologies;
+  const { technologies } = projects[index - 1];
+
+  function clearPopupSkills() {
+    popupSkills.innerHTML = '';
+  }
 
+  clearPopupSkills();
+  technologies.forEach((technology) => {
```
Comment (LionRouge1) [inline main.js]: - [ ] As suggested by the previous reviewer, kindly make sure the popup menu is display dynamically according to the project information on which the use has clicked.

- [ ] Kindly reduce the `height` of the popup menu to follow the Figma design 
### Your design :point_down: 
Screenshot from 2023-07-29 00-13-23
### What it is supposed to look like :point_down: 
Screenshot from 2023-04-19 13-38-59

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @donice, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

## Suggested changes :recycle: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @donice , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that doesn't have linters passing please kindly find the errors and fix them 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @donice , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that do not have linters running 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css
Comment (franclobo) [inline style.css:1]: - [ ] I recommend doing the following changes in the stiles to be similar to the Figma template and complete the last CR requirement 👇🏽 

> Kindly reduce the height of the popup menu to follow the Figma design

**Current code**

```css
#popupContainer {
  display: none;
  position: fixed;
  top: 0;
  left: 100%;
  width: 100%;
  min-height: 100vh;
  z-index: 98;
  background: var(--neutrals-n-50, #c1c7d04f);
  backdrop-filter: blur(10px);
  overflow-y: hidden;
  padding: 18px;
}
```

**Changes suggested**

```css
#popupContainer {
  display: none;
  position: fixed;
  top: 0;
  left: 100%;
  width: 100%;
  height: 100vh;
  z-index: 98;
  background: var(--neutrals-n-50, #c1c7d04f);
  backdrop-filter: blur(10px);
  overflow-y: scroll;
  padding: 18px;
}
```

**Note** Review that the mobile version works as expected, the popup window should scroll to allow the user to see the content.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @donice, 

# Required Changes ♻️

istockphoto-1079018918-612x612

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️
- Descriptive PR ✔️
- Correct Github flow ✔️
- Good UI ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @donice  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Pop-up works as expected💯
-  Excellent communication with the reviewer👍
-  Correct Github flow🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**